### DISCUSSION TITLE
Welcome to Plasma Discussions!

### DISCUSSION THREAD

**Discussion by dpogue** [H-uru/Plasma#735] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and plan work relating to the Plasma engine. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and proposals.
  * Engage with other community members.
  * Welcome others and are open-minded.

 This is very much an experiment to see if GitHub Discussions are a good fit for what we're hoping to achieve 😅

**Comment by Hoikas** [comment]
Welcome to PlasmaCom. This is PlasmaCom. Welcome.

**Reply by Jahmen-Myst** [reply]
How did your plasma.com workout? I went to the http//:www.plasma.com site and it was in German>
Was it to rework the plasma code for the Marker Editor?

**Reply by dgelessus** [reply]
Hoikas' comment was just a joke (see https://en.wikipedia.org/wiki/Zombo.com). The real website for H'uru's Plasma engine is at https://h-uru.github.io/Plasma/.

As you've already found out, https://plasma.com/ belongs to a German company (Diener electronic) that manufactures plasma-related machines. They have an [English website](https://plasma.com/en/) too, if you're really interested ;)
